models/users.js

```diff
@@ -69,7 +71,12 @@ const addOrUpdate = async (userData, userId = null) => {
     userData.roles = { archived: false, in_discord: false };
     userData.incompleteUserDetails = true;
     const userInfo = await userModel.add(userData);
-    return { isNewUser: true, userId: userInfo.id, incompleteUserDetails: true, updated_at: Date.now() };
+    return {
+      isNewUser: true,
+      userId: userInfo.id,
```

Comment:
Why is this being added here?

Reply:
Its the same returning from here, just changed the formatting. Should I keep same as it before?

Reply:
yes @ankitbajpai65 keep it same, this confuses other people 😅😅😅